docs/source/release_notes/release-notes.rst

```diff
@@ -80,6 +80,11 @@ develop
     *    - |devbreak|
          - The persistent lock release endpoint has now been renamed to ``releaseBackupLock`` since it is currently only supposed to be used for the backup lock.
 
+    *    - |deprecated|
+         - ``PaxosLeaderElectionService`` is now deprecated, to mitigate risks of users supplying parameters in the wrong order.
```

Comment:
You mean that the constructor is deprecated - this makes it sound like the whole service has been deprecated.